TidepoolKit/Model/TInfo.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  TInfo.swift
+//  TidepoolKit
+//
+//  Created by Rick Pasetto on 9/7/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+public struct TInfo: Codable, Equatable {
```

Comment:
At this level (which isn't supposed to know anything about Loop or necessarily what any of the fields mean or do not mean; this library is agnostic), I think this should be trimmed down to:
```
public struct TInfo: Codable, Equatable {
    public var versions: Versions?

    public struct Versions: Codable, Equatable {
        public var uploaderMinimum: String?
        public var loop: Loop?

        public struct Loop: Codable, Equatable {
            public var minimumSupported: String?
            public var criticalUpdateNeeded: [String]?
        }
    }
}
```
The rest of the logic that is specific to Loop should move into LoopKit (not TidepoolService, which is also supposed to be generic; though there may be a translation layer there from the TidepoolKit type (which includes a bunch of unnecessary stuff like uploaderMinimum) to a LoopKit type which would just have the Loop version info and none of the wrappers) which will call this API.

BTW, structs within other structs do not use the "T" prefix.

Reply:
I hear what you're saying, but, there's a "Loop" right there in the struct...and the nature of the API itself is to talk about app versions.  Not sure you can avoid "loop-ness".  But I'll make the change you suggest.